src/main-process/parse-command-line.js

```diff
@@ -130,21 +128,8 @@ module.exports = function parseCommandLine (processArgs, initialResourcePath) {
     }
   }
 
-  if (args['resource-path']) {
+  if (args['resource-path'] || test) {
     devMode = true
-    devResourcePath = args['resource-path']
-  }
-
-  if (test) {
-    devMode = true
-  }
-
-  if (devMode) {
-    resourcePath = devResourcePath
-  }
```

Comment:
Is this conditional (assigning `resourcePath = devResourcePath` in dev mode) not needed anymore?

Reply:
It shouldn't be needed since it's being done here: https://github.com/atom/atom/blob/bb1f7b0a2fd8eb37e94843f6ef7337a61f5f3d83/src/main-process/main.js#L50

Reply:
Oh yeah, sweet.